trestle/core/commands/author/profile.py

```diff
@@ -470,14 +470,14 @@ def resolve_profile(
             trestle_root: Root directory of the trestle workspace
             profile_path: Path of the profile json file
             catalog_name: Name of the resolved profile catalog
-            show_values: If true, show values of parameters in prose rather than [Assignment:] form
+            show_values: If true, show values of parameters in prose rather than original {{}} form
 
         Returns:
             0 on success and raises exception on error
         """
         if not profile_path.exists():
             raise TrestleNotFoundError(f'Cannot resolve profile catalog: profile {profile_path} does not exist.')
-        param_rep = ParameterRep.VALUE_OR_LABEL_OR_CHOICES if show_values else ParameterRep.ASSIGNMENT_FORM
+        param_rep = ParameterRep.VALUE_OR_LABEL_OR_CHOICES if show_values else ParameterRep.LEAVE_MOUSTACHE
```

Comment:
I think this should be an argument to provide to resolve_profile, in case anyone needs assignment representation in the future